pages/1.11/release-notes/1.11.11/index.md

```diff
@@ -0,0 +1,115 @@
+title: Release Notes for 1.11.11
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.11
+---
+
+DC/OS Version 1.11.11 was released on June 5, 2019.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.11/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="http://downloads.mesosphere.com/dcos-enterprise/stable/1.11.11/dcos_generate_config.ee.sh"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.11 includes the following components:
+- Apache Mesos 1.5.4 [change log](https://github.com/apache/mesos/blob/6a8b61b59498dcea64d9380a233d517ce65dc2ef/CHANGELOG).
+- Marathon 1.6.585 [change log](https://github.com/mesosphere/marathon/blob/3e0898228572e5c758c4ae17838fc5a9b0459e92/changelog.md).
+- Metronome 0.6.18 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.5).
+
+# Release Summary
+
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment. 
+
+# Issues Fixed in DC/OS 1.11.11
+The issues that have been fixed in DC/OS 1.11.11 are grouped by feature, functional area, or component. Most change descriptions include one or more issue tracking identifiers for reference.
+
+
+## CLI 
+- DCOS-42928, DCOS_OSS-5105 - Use `docker ps` instead of `docker up` in diagnostics, since `docker up` is an invalid Docker command. 
+- DCOS_OSS-1502 - There were couple issues caused by unused volumes for removed containers. It is recommended to use docker prune commands to avoid the occurrence of such issues. This release uses `docker-gc` command to remove the unused volumes.
+
+## Exhibitor
+- DCOS-51751 - The configuration parameters `aws_secret_access_key` and `exhibitor_azure_account_key` for Exhibitor are now marked as secret. They will not be revealed in `user.config.yaml` on cluster nodes, but will from now on appear only in `user.config.full.yaml` which has stricter read permissions and is not included in DC/OS diagnostics bundles.
+
+## GUI
+- DCOS-54087 - In the previous releases, UI overrides `concurrencyPolicy` field in schedule to always be `ALLOW`. In this release, the `concurrencyPolicy` field is fixed for UI to keep the value intact, thus allowing other concurrency policies other than `ALLOW`.
+
+## Installation
+- DCOS-15890 - The pre-flight check on advanced installer shows misleading information. This release improves the context of an error message, in case Docker is not running at the start of installation.
+- DCOS-53077 - Fixed a number of issues that caused some DC/OS components to crash when `/tmp` is mounted with the `noexec` option.
+
+## MARATHON
+- MARATHON-8562 - Some HTTP endpoints are blocked with `RestResource.result` even though the non-blocking async replies with `sendResponse(asyncResponse)`. This release removes the `RestResource.result` completely and adapts all endpoints that return a failure.
```

Comment:
```suggestion
```

Reply:
@anithamuthu79 this was an internal improvement and doesn't really lead to any user facing change.